### PR TITLE
Update favicon to higher resolution

### PR DESCRIPTION
**Pull request by connorbey13**
Now that we know what assets to use, this is what was done for this PR:

- Used the image at [https://github.com/marcoroth/turbo_power/blob/main/assets/icon.png](https://github.com/marcoroth/turbo_power/blob/main/assets/icon.png) and filled the bolt in white and rounded the corners.
- Generated higher resolution favicons and webmanifest that support multiple browsers at [https://favicon.io/favicon-converter/](https://favicon.io/favicon-converter/).
- Replaced favicon text in head with text from the generator.
- Removed unneeded file `apple-touch-icon-precomposed.png`.

### CODE REVIEW

File: public/android-chrome-512x512.png
Comment (marcoroth) [inline public/android-chrome-512x512.png:1]: Is there a way to make those edges smoother?

**Review by marcoroth — CHANGES_REQUESTED** [review]
Thanks for taking this on again @connorbey13!

File: public/android-chrome-512x512.png
Comment (connorbey13) [reply public/android-chrome-512x512.png:1]: Done! Found a much better way to do that than I did originally.

**Review by marcoroth — APPROVED** [review]
Thank you @connorbey13! 🙏🏼

**Pull request merged**